lang/go/bindings/examples/sampling/README

```diff
@@ -0,0 +1,17 @@
+Brief Description about Sampling Go Apllication
```

Comment:
Convert this file from a README to a doc.go file. This allows us to use the document tools to display this type of information, which is something needed for other Go code and will create an issue. Make sure the copyright notice is correct with the current year only.